lib/machine-loader.js

```diff
@@ -83,28 +85,29 @@ function getMachineNames (config) {
  * @returns {string} machine name
  */
 function getMachineName (machineId) {
-  const sql = 'select * from devices where device_id=$1'
+  const sql = 'SELECT * FROM devices WHERE device_id=$1'
   return db.oneOrNone(sql, [machineId])
     .then(it => it.name)
 }
 
 function getMachine (machineId) {
-  const sql = 'select * from devices where device_id=$1'
+  const sql = 'SELECT * FROM devices WHERE device_id=$1'
   return db.oneOrNone(sql, [machineId]).then(res => _.mapKeys(_.camelCase)(res))
 }
 
 function renameMachine (rec) {
-  const sql = 'update devices set name=$1 where device_id=$2'
+  const sql = 'UPDATE devices SET name=$1 WHERE device_id=$2'
   return db.none(sql, [rec.newName, rec.deviceId])
 }
 
 function resetCashOutBills (rec) {
-  const sql = 'update devices set cassette1=$1, cassette2=$2 where device_id=$3'
-  return db.none(sql, [rec.cassettes[0], rec.cassettes[1], rec.deviceId])
+  const detailB = notifierUtils.buildDetail({deviceId: rec.deviceId})
```

Comment:
Format like standardJS

```suggestion
  const detailB = notifierUtils.buildDetail({ deviceId: rec.deviceId })
```